test/test_modules.py

```diff
@@ -932,6 +931,47 @@ def _to(m, set_grad=False):
                         self.assertTrue(all(a == b for a, b in zip(g_ids_before, g_ids_after)))
 
 
+    @modules([module for module in module_db if not module.is_lazy], allowed_dtypes=[torch.float32])
+    @parametrize('swap', [True, False])
+    @wrapSwapTensorsTest()
+    def test_to_empty(self, device, dtype, module_info, swap, training):
+        module_cls = module_info.module_cls
+
+        with torch.device("meta"):
+            module_inputs = module_info.module_inputs_func(module_info, device=None, dtype=dtype,
+                                                           requires_grad=False, training=training)
+
+        torch.__future__.set_swap_module_params_on_conversion(swap)
```

Comment:
Use a try/finally to make sure this is reset properly even when an error happens

Reply:
This is handled by `@wrapSwapTensorsTest()` on 936 :D

Reply:
Ho yeah I forgot that.